Libplanet.Net/Consensus/ConsensusContext.cs

```diff
@@ -452,20 +452,28 @@ private Context CreateContext(long height)
         {
             // blockchain may not contain block of Height - 1?
             ValidatorSet validatorSet;
-            if (_blockChain.Tip.ProtocolVersion
-                < BlockMetadata.SlothProtocolVersion)
+            if (_blockChain.Tip.ProtocolVersion < BlockMetadata.SlothProtocolVersion)
             {
                 validatorSet = _blockChain
                     .GetWorldState(_blockChain[Height - 1].StateRootHash)
```

Comment:
The operation would be same, but using parameter would be better for the meaning, IMO.
Or, we have to remove the parameter of this method I think.
```suggestion
                    .GetWorldState(_blockChain[height - 1].StateRootHash)
```

Reply:
Ah, now I remember.
Missed it's also called on https://github.com/planetarium/libplanet/blob/cc268b022eb3569d2056d29174a51efaeb408d18/Libplanet.Net/Consensus/ConsensusContext.cs#L269
Seems like there are no validator replacing logic, and pre-generation of higher context would raises some bugs, right?